extensions/gdx-freetype/src/com/badlogic/gdx/graphics/g2d/freetype/FreeType.java

```diff
@@ -341,6 +342,186 @@ public int getCharIndex(int charCode) {
+			return getNumDeisgns(address);
+		}
+
+		private static native int getNumNamedStyles(long mmVar); /*
+			return ((FT_MM_Var *) mmVar)->num_namedstyles;
+		*/
+
+		public int getNumNamedStyles() {
+			return getNumNamedStyles(address);
+		}
+
+		private static native long[] getAxis(long mmVar); /*
+			FT_MM_Var* v = ((FT_MM_Var *) mmVar);
+			FT_Var_Axis* axis = v->axis;
+			jlongArray arr = env->NewLongArray(v->num_axis);
+			jlong* ptr = env->GetLongArrayElements(arr, NULL);
+			for (int i = 0; i < v->num_axis; i++) {
+				ptr[i] = (jlong) &axis[i];
+			}
+			env->ReleaseLongArrayElements(arr, ptr, 0);
+			return arr;
+		*/
+
+		public VarAxis[] getAxis() {
+			long[] axisPtrs = getAxis(address);
+			VarAxis[] axis = new VarAxis[axisPtrs.length];
+			for (int i = 0; i < axisPtrs.length; i++) {
+				axis[i] = new VarAxis(axisPtrs[i]);
+			}
+			return axis;
+		}
+
+		private static native long[] getNamedStyle(long mmVar); /*
+			FT_MM_Var* v = ((FT_MM_Var *) mmVar);
+			FT_Var_Named_Style* namedStyle = v->namedstyle;
+			jlongArray arr = env->NewLongArray(v->num_namedstyles);
+			jlong* ptr = env->GetLongArrayElements(arr, NULL);
+			for (int i = 0; i < v->num_namedstyles; i++) {
+				ptr[i] = (jlong) &namedStyle[i];
+			}
+			env->ReleaseLongArrayElements(arr, ptr, 0);
+			return arr;
+		*/
+
+		public VarNamedStyle[] getNamedStyle() {
+			long[] namedStylePtrs = getNamedStyle(address);
+			VarNamedStyle[] namedStyle = new VarNamedStyle[namedStylePtrs.length];
+			for (int i = 0; i < namedStylePtrs.length; i++) {
+				namedStyle[i] = new VarNamedStyle(namedStylePtrs[i]);
+			}
+			return namedStyle;
+		}
+
+	}
+
+	public static class VarAxis extends Pointer {
+
+		public VarAxis(long address) {
+			super(address);
+		}
+
+		private static native long getMinimum(long varAxis); /*
+			return (jlong) (((FT_Var_Axis *) varAxis)->minimum);
+		*/
+
+		public long getMinimum() {
+			return getMinimum(address);
+		}
+
+		private static native long getDef(long varAxis); /*
+			return (jlong) (((FT_Var_Axis *) varAxis)->def);
+		*/
+
+		public long getDef() {
+			return getDef(address);
+		}
+
+		private static native long getMaximum(long varAxis); /*
+			return (jlong) (((FT_Var_Axis *) varAxis)->maximum);
+		*/
+
+		public long getMaximum() {
+			return getMaximum(address);
+		}
+
+		private static native long getTag(long varAxis); /*
+			return (jlong) (((FT_Var_Axis *) varAxis)->tag);
+		*/
+
+		public long getTag() {
+			return getTag(address);
+		}
+
+		private static native int getStrid(long varAxis); /*
+			return (jint) (((FT_Var_Axis *) varAxis)->strid);
+		*/
+
+		public int getStrid() {
+			return getStrid(address);
+		}
+
+	}
+
+	public static class VarNamedStyle extends Pointer {
+
+		public VarNamedStyle(long address) {
+			super(address);
+		}
+
+		// No getCoords
+
+		private static native int getStrid(long varNamedStyle); /*
+			return (jint) (((FT_Var_Named_Style *) varNamedStyle)->strid);
+		*/
+
+		public int getStrid() {
```

Comment:
It might be worthwhile (if this faces the user at all) to just put a quick note with what the docs say for a parameter or just link to the FreeType docs at the top of the file, in a comment. https://freetype.sourceforge.net/freetype2/docs/reference/ft2-multiple_masters.html I did a double-take at "getStrid()" because I assumed it was a typo of "stride" -- I was surprised to learn that's the name used by FreeType, though the variable apparently doesn't matter to how FreeType works.